projects/bp-gallery/src/components/views/start/StartView.tsx

```diff
@@ -55,6 +55,15 @@ const StartView = () => {
               <p>{t('startpage.welcome-text')}</p>
             </div>
             <DailyPicture />
+            <div className='flex place-content-center gap-2 m-4'>
+              <PrimaryButton
+                text='Geoguessor'
```

Comment:
Firstly, is this misspelled intentionally to avoid copyright issues with GeoGuesser? Secondly, maybe put this inside `de.json`?